EIPS/eip-7540.md

```diff
@@ -238,6 +238,13 @@ MUST be emitted when a redemption Request is submitted using the `requestRedeem`
       type: uint256
 ```
 
+### ERC-165 support
```

Comment:
I think this will fail linting if the link isn't in the header

Reply:
Updated!